plutus-pab/src/Plutus/PAB/LocalCluster/Run.hs

```diff
@@ -279,7 +287,7 @@ launchPAB userContractHandler
                     , pscSlotConfig = slotConfig
                     , pscKeptBlocks = securityParam
                     }
-                , dbConfig = def{dbConfigFile = T.pack (dir </> "plutus-pab.db")}
+                , dbConfig = def
```

Comment:
It's breaking the behaviour — the path should be preserved.

Reply:
I also agree. Already fixed.